### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomSmile**
DEMO LINK: https://RomSmile.github.io/layout_stop-watch/

### CODE REVIEW

File: src/styles/untils/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+$height-of-minute-pointer:
+    calc(#{$wrapper-diametr} / 4);
+$height-of-second-pointer:
+    calc(#{$wrapper-diametr} / 3);
+$pointer-left-position:
+    calc(#{$wrapper-diametr} / 2 - #{$width-of-pointers} / 2);
+$pointer-minute-top-position:
+    calc(#{$wrapper-diametr} / 2 - #{$height-of-minute-pointer});
+$pointer-second-top-position:
+    calc(#{$wrapper-diametr} / 2 - #{$height-of-second-pointer});
```
Comment (DmytroMykoliv) [inline src/styles/untils/_variables.scss:15]: use variables without calc , like this -->   $circle-position: $wrapper-diametr / 2 - $circle-diametr / 2;

File: src/styles/reset.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+}
```
Comment (DmytroMykoliv) [inline src/styles/reset.scss:7]: в body mast be margin: 0; , not padding

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page">
+    <div class="page__wrapper">
+      <div class="page__circle"></div>
+      <div class="page__second-pointer"></div>
+      <div class="page__minute-pointer"></div>
+    </div>
```
Comment (DmytroMykoliv) [inline src/index.html:15]: make page__circle the last block and it will not be under the arrows on screen

**Review by DmytroMykoliv — APPROVED** [review]
Everything is fine except the little things

File: src/styles/untils/_variables.scss (hunk 1 above)
Comment (RomSmile) [reply src/styles/untils/_variables.scss:15]: Я на вский случай, чтобы они работали)

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**